petastorm/spark/spark_dataset_converter.py

```diff
@@ -1,21 +1,39 @@
 from petastorm import make_batch_reader
+from petastorm.fs_utils import FilesystemResolver
+from petastorm.reader import normalize_data_url
 from petastorm.tf_utils import make_petastorm_dataset
 from pyspark.sql.session import SparkSession
 
 import atexit
 import os
+import pyarrow
 import shutil
 import threading
 import uuid
 
-DEFAULT_CACHE_DIR = "/tmp/spark-converter"
+DEFAULT_CACHE_DIR = "file:///tmp/spark-converter"
 ROW_GROUP_SIZE = 32 * 1024 * 1024
 
 
 def _get_spark_session():
     return SparkSession.builder.getOrCreate()
 
 
+def _delete_cache_data(dataset_url):
```

Comment:
swallow exception raised from `_delete_cache_data`,
and add a warning log if capture exception.